presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -429,6 +433,54 @@ public void testSplitsGenerationWithAbortedTransactions()
         }
     }
 
+    @Test
+    public void testFullAcidTableWithOriginalFilesFails()
+            throws Exception
+    {
+        java.nio.file.Path tablePath = Files.createTempDirectory(UUID.randomUUID().toString());
```

Comment:
`Files.createTempDirectory("")` (or `"TestBackgroundHiveSplitLoader"`) would be just fine, no need to invoke UUID here

(https://github.com/prestosql/presto/pull/2295)

Reply:
changing as per the PR to "TestBackgroundHiveSplitLoader"